cpu/doc.txt

```diff
@@ -11,6 +11,6 @@
  * @brief CPU specific implementations
  *
  * This module contains all CPU specific source files. In case of multiple CPUs
- * sharing the same architecture, the implementation is split in a cpu specific
- * and a architecture specific part (eg. arm-common and lpc2387).
+ * sharing the same architecture, the implementation is split into several CPU
+ * specific parts and an architecture part (e.g. arm-common and lpc2387).
```

Comment:
The directory is called `arm7_common` since 20ce2aee8bc22e200ccef0d5f5845fdd9ec0e9b9 and before that was called `arm_common` since 91ae1eb6fdba1ee2048